packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -46,7 +48,10 @@ class Chat extends Widget<Properties> {
 
     super._initMarkup();
 
-    this._renderHeader();
+    if (shouldHeaderBeRendered) {
```

Comment:
It is strange to see a magic variable used to prevent the chat header from rendering. It's better to check that the default value has not been changed; that should be sufficient. We discussed this way.

This approach also allows us to test it in the usual way without using global variables or QUnit.skip.

Reply:
You are proposing an implementation that is too complex.

Reply:
[1b9f995421176c5da1ede84866276a1288870383]

Reply:
Also, now, we must support all scenarios with title change in runtime

Reply:
[b2a70f90cdda0c35379a3881f4e03131bf01d0a8]